### PR TITLE
Resolve few issue about class

### PR DESCRIPTION
**Pull request by ENvironmentSet**
resolves: #131
[proposal/decorator#128](tc39/proposal-decorators#128)

### CODE REVIEW

File: spec.html [hunk 1]
```diff
@@ -286,8 +286,7 @@ <h1>Runtime Semantics: ClassElementEvaluation</h1>
       ClassElement : `static` MethodDefinition
   </emu-grammar>
   <emu-alg>
-    1. Perform ? PropertyDefinitionEvaluation with parameters _object_ and _enumerable_.
-    1. Return an empty List.
+    1. Return PropertyDefinitionEvaluation with parameters _object_ and _enumerable_.
```
Comment (littledan) [inline spec.html]: Hmm, I don't think this value will be useful the way you might be thinking. We need to somehow omit methods from the field list, don't we?

Comment (ENvironmentSet) [reply spec.html]: ah, I missed that part, currently we don't omit methods. I see, I will find better way like to filter methods from the element list. and so on :)

Comment (ENvironmentSet) [reply spec.html]: @littledan 
I think this proposal needs descriptor to describe class field and method that has compatible with proposal-private-methods's one and proposal-decorator's one.

actually, we have one. but it's for only class field. So, It cause discrepancies of `ClassElementEvaluation` (when evaluate class field, returns Record, but others are not).

Comment (littledan) [reply spec.html]: Right, so it's either changing the convention in all of the specs or none of them. I would be fine either way.

**Comment by littledan** [comment]
I believe this issue has been refactored away, so closing the PR.

**Pull request closed without merging**